configs/sites/hera/packages.yaml

```diff
@@ -1,30 +1,31 @@
 packages:
   all:
-    compiler:: [intel@2021.3.0]
+    compiler:: [intel@2021.5.0]
     providers:
-      mpi:: [intel-oneapi-mpi@2021.3.0]
+      mpi:: [intel-oneapi-mpi@2021.5.1]
   mpi:
     buildable: False
   intel-mpi:
-    buildable: False
     externals:
     - spec: intel-mpi@2018.0.4
       modules:
       - impi/2018.0.4
-    - spec: impi@2019.0.5
-      modules:
-      - impi/2019.0.5
-    - spec: impi@2020
+    - spec: intel-oneapi-mpi@2021.5.1
       modules:
-      - impi/2020
-    - spec: intel-oneapi-mpi@2021.3.0
-      modules:
-      - impi/2021.3.0
+      - impi/2022.1.2
   openmpi:
     externals:
     - spec: openmpi@3.1.4%gcc@9.2.0~cuda+cxx+cxx_exceptions~java~memchecker+pmi+pmix~sqlite3+static~thread_multiple~wrapper-rpath fabrics=mxm schedulers=slurm
       modules:
       - openmpi/3.1.4
+  python:
+    buildable: False
+    externals:
+    - spec: python@3.9.7+bz2+ctypes+dbm+ensurepip+lzma+nis+pyexpat+pythoncmd~readline+sqlite3+ssl+tix+tkinter+uuid+zlib
+      modules:
+      - intelpython/2022.1.2
+      prefix: /apps/oneapi/intelpython/python3.9
```

Comment:
I thought we wanted to use a different Python?

Reply:
Yes, hera is still in the works  (see "still todo section"). Let me push what I have so far.

Reply:
Ok, I pushed my updates.